resources/js/components/UserEditForm.vue

```diff
@@ -273,6 +303,14 @@ export default {
           } else {
             this.feedback = 'Saved!';
           }
+
+          if (this.user.autodesk_email !== this.AutodeskEmailInDatabase && this.user.autodesk_email && this.user.autodesk_email.length > 0) {
+            this.AutodeskEmailInDatabase = this.user.autodesk_email;
+            this.feedback = 'Saved! Look out for an email from Autodesk in the next few minutes.'
+          } else {
+            this.feedback = 'Saved!';
+          }
+
```

Comment:
The feedback for the clickup field will always be overwritten by the autodesk feedback. For instance, if you set a clickup email and not an autodesk email, `this.feedback` will be set on line 302 and then overwritten by line 311. The feedback from both should be merged.